### DISCUSSION TITLE
Purpose of Discussion board

### DISCUSSION THREAD

**Discussion by ted-miller** [Yaskawa-Global/YMConnect#1] [Announcements]
We’re using Discussions as a place to provide support for the YMConnect library.

Please use this forum to:
- Ask questions on usage of the API's
- Request technical support with using the library
- Request features to improve functionality

For sensitive matters that you wish to keep away from a public forum, please contact techsupport@motoman.com.

**Comment by RubyAI132** [comment]
Hello I want to follow the quick start  tutorial  
I can't find the C++ header files for Windows 

Is there a download source

**Reply by ted-miller** [reply]
Hello.
The header and library files are available on the Github [Releases page](https://github.com/Yaskawa-Global/YMConnect/releases).

Documentation on the API is available in the [Motoman Developer Portal](https://developer.motoman.com/en/YMConnect).
